Makefile

```diff
@@ -76,6 +76,11 @@ shell:
 	-v ${PWD}:/go/src/github.com/kris-nova/kubicorn \
 	--rm ${SHELL_IMAGE} /bin/bash
 
+lint:
+	which golint > /dev/null || go get -u github.com/golang/lint/golint...
```

Comment:
It's not the same situation. If you take a closer look, you'll see that you have something such as `go-bindata/...`, which expands and installs all the packages under that directory.
That's not the case with `golint`. Inside `github.com/golang/lint/golint` you don't have any other package beside `golint`, so you just need to install that one. 
I hope you understand better how it works. :)